rhel/ovn-fedora.spec.in

```diff
@@ -264,11 +264,14 @@ fi
 %preun central
 %if 0%{?systemd_preun:1}
     %systemd_preun ovn-northd.service
+    %systemd_preun ovn-ic.service
 %else
     if [ $1 -eq 0 ] ; then
         # Package removal, not upgrade
         /bin/systemctl --no-reload disable ovn-northd.service >/dev/null 2>&1 || :
         /bin/systemctl stop ovn-northd.service >/dev/null 2>&1 || :
+        /bin/systemctl --no-reload disable ovn-ic.service >/dev/null 2>&1 || :
+        /bin/systemctl stop ovn-ic.service >/dev/null 2>&1 || :
```

Comment:
When we split OVN from OVS, we had to do some special handling from upgrading and hence had to handle certain things in %preun and other macros.

Since ovn-ic was added after OVN split, I don't think we need to handle for ovn-ic here.

Can you please try it out removing this special handling for ovn-ic here ? And see if it works fine ?
If it works fine, I'd suggest to remove these and update the PR.

Thanks
Numan

Reply:
Otherwise the PR looks goo to me.

Reply:
Sorry, I think I didn't get your idea. Why not disable and stop ovn-ic.service before ovn-central package removal?

Reply:
Ok. Sorry. I think I got confused.